spec/models/user_spec.rb

```diff
@@ -138,4 +140,25 @@
       expect(user.display_name).to eq('hasso01')
     end
   end
+
+  context 'with jitsi calls' do
+    let(:jitsi_calls) { FactoryBot.create_list :jitsi_call, 2 }
+
+    before do
+      user.save
```

Comment:
We wouldn't need this if we use build in the next line, would we?

Reply:
No idea

Reply:
It does not work because we check the number of call participants in the database in `destroys all participants when destroyed`